VMBackup/main/fsfreezer.py

```diff
@@ -147,7 +150,14 @@ def __init__(self, patching, logger, hutil):
         self.getLockRetry = 0
         self.maxGetLockRetry = 5
         self.safeFreezelockFile = None
-
+    
+    def isFileExists(self, relative_path):
+        full_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), relative_path)
+        self.logger.log("path "+ str(full_path))
+        self.logger.log("path exists " + str(os.path.exists(full_path)))
```

Comment:
we can introduce a local variable to keep the value instead of calling os.path.exists multiple times.

self.file_exists = os.path.exists(full_path)
self.logger.log("path " + str(full_path) + ". Found the file: " + fileExists)